centos-quarkus-native-s2i/README.md

```diff
@@ -11,12 +11,23 @@
 
 ### OpenShift Build
 
+You can locally build this S2I Builder image inside your OpenShift cluster like this:
+
     oc new-build https://github.com/quarkusio/quarkus.git --context-dir=docker/centos-graal-maven-s2i --name quarkus-native-s2i
     oc logs -f bc/quarkus-native-s2i
 
+It is also available on https://quay.io/repository/quarkus/centos-quarkus-native-s2i.
+
 ### OpenShift Use
 
+    oc new-app quay.io/quarkus/centos-quarkus-native-s2i~https://github.com/quarkusio/quarkus-quickstarts --context-dir=getting-started --name=getting-started-native
+
+or, to use the image locally built above:
+
     oc new-app quarkus-native-s2i~https://github.com/quarkusio/quarkus-quickstarts --context-dir=getting-started --name=getting-started-native
```

Comment:
```suggestion
   oc new-app https://github.com/quarkusio/quarkus-quickstarts --context-dir=getting-started --name=getting-started-native --image-stream="quarkus-native/centos-quarkus-native-s2i:latest"    
```

Reply:
That command fails for me with:

```
➜  oc new-app quarkus-native-s2i~https://github.com/quarkusio/quarkus-quickstarts --context-dir=getting-started --name=getting-started-native

error: only a partial match was found for "quarkus-native-s2i": "quarkus-native/centos-quarkus-native-s2i:latest"

The argument "quarkus-native-s2i" only partially matched the following Docker image, OpenShift image stream, or template:

* Image stream "centos-quarkus-native-s2i" (tag "latest") in project "quarkus-native"
  Use --image-stream="quarkus-native/centos-quarkus-native-s2i:latest" to specify this image or template
```

Reply:
> That command fails for me with:

@gastaldi that's curious, it worked fine for me locally.. did you perhaps run `oc new-build https://github.com/quarkusio/quarkus.git` in a different `oc project` namespace/context (the `quarkus-native`) than where you do `oc new-app ... quarkus-quickstarts` ? What's what this error message seems to tell us.

Your Suggested Change `-image-stream="quarkus-native/centos-quarkus-native-s2i:latest` would only work if you did that, but not in the general case - that `quarkus-native` is "arbitrary" (not suggested in the previous step) - am I making any sense?

Does it work if you just do `oc new-app quay.io/quarkus/centos-quarkus-native-s2i~https://github.com/quarkusio/quarkus-quickstarts --context-dir=getting-started --name=getting-started-native` and therefore use the container what is available on https://quay.io/repository/quarkus/centos-quarkus-native-s2i, instead of locally building this S2I Builder image inside your OpenShift cluster yourself?

Reply:
@vorburger 
```oc new-app quay.io/quarkus/centos-quarkus-native-s2i~https://github.com/quarkusio/quarkus-quickstarts --context-dir=getting-started --name=getting-started-native``` works for me

Reply:
But the output is:

```
  | 2019-03-25 21:59:52,866 ERROR [null] (main) Failed to publish log record (OPEN_FAILURE[4]): Failed to set log file: java.io.FileNotFoundException: quarkus.log
-- | --
  | at com.oracle.svm.core.posix.PosixUtils.fileOpen(PosixUtils.java:234)
  | at java.io.FileOutputStream.open(FileOutputStream.java:618)
  | at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
  | at org.jboss.logmanager.handlers.FileHandler.setFile(FileHandler.java:148)
  | at io.quarkus.runtime.logging.LoggingSetupTemplate.initializeLogging(LoggingSetupTemplate.java:71)
  | at io.quarkus.deployment.steps.LoggingResourceProcessor$setupLoggingRuntimeInit2.deploy(Unknown Source)
  | at io.quarkus.runner.ApplicationImpl1.doStart(Unknown Source)
  | at io.quarkus.runtime.Application.start(Application.java:93)
  | at io.quarkus.runtime.Application.run(Application.java:198)
  | at io.quarkus.runner.GeneratedMain.main(Unknown Source)
  |  
  | 2019-03-25 21:59:52,873 INFO  [io.quarkus] (main) Quarkus 0.12.0 started in 0.009s. Listening on: http://[::]:8080
  | 2019-03-25 21:59:52,873 INFO  [io.quarkus] (main) Installed features: [cdi, resteasy]
```

Probably it is missing the `quarkus.log.file.enabled=false` (or something similar) property?

Reply:
@gastaldi do you have time to verify if #7 fixes this problem? Because I can't reproduce it (weird).